### PR TITLE
Built different sections

### PR DESCRIPTION
**Pull request by tarikbouari**
In this project I:

- created a project section with a grid and flex-box on the home page
- created about section and list items 
- fixed linters and updated my code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,6 +35,149 @@ <h1 class="name">I’m Tarik Glad to see you!</h1>
...
+            </div>
+        </div>
+     </section>
+
+     <!--About me Section-->
+    <section class="about-me">
+        <div class="personal-info">
+            <h2>About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="blue text">LET’S CONNECT</p>
```
Comment (cwaku) [inline index.html]: - [ ] It is required that you use a header tag `hx` for the "Let's connect" subtitle. Kindly fix that.

File: index.html [hunk 2]
```diff
@@ -35,6 +35,149 @@ <h1 class="name">I’m Tarik Glad to see you!</h1>
...
+                                <p>JavaScript</p>
+                            </div>
+                            <div class="html">
+                                <img src="images/html.png" class="img-lang" alt="html">
+                                <p>HTML</p>
+                            </div>
+                            <div class="css">
+                                <img src="images/css.png" class="img-lang" alt="">
+                                <p>CSS</p>
+                            </div>
```
Comment (cwaku) [inline index.html]: - [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. I suggest you fix that

File: index.html [hunk 3]
```diff
@@ -35,6 +35,149 @@ <h1 class="name">I’m Tarik Glad to see you!</h1>
...
+                <li>
+                    <div class="lang">
+                        <h3>Frameworks</h3>
+                        <img src="images/angle-right.svg" class="arrow icon" alt="arrow-right">
+                    </div>
+                </li>
+                <li>
+                    <div class="lang">
+                        <h3>Skills</h3>
+                        <img src="images/angle-right.svg" class="arrow icon" alt="arrow-down">
```
Comment (cwaku) [inline index.html:174]: - [ ] It happens that your arrows here are not displaying as it shows on the figma design. I looked into it and found that there's a padding bottom being applied to them with CSS. Kindly fix that. 

##### Figma Design
image

##### Your Design
image

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @tarikbouari 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️  but you are almost there! 🏃🏾 

## Highlights
- [x] Good commit history 👍🏾 
- [x] Meaningful PR title and description 💯 
- [x] Correct use of GitHub flow 🥇 
- [x] Linters passed 👍🏾 
- [x] Professional repo 💯 

## Required Changes ♻️
It's OK to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @tarikbouari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

 Good commit history :heavy_check_mark: 
 Good Pull Request title and description :heavy_check_mark: 
 Good use of github workflow :heavy_check_mark: 
 No Linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**